src/components/ImagesCarousel.js

```diff
@@ -1,24 +1,31 @@
 import PropTypes from 'prop-types';
-import Carousel from 'react-native-snap-carousel';
-import React, { useCallback, useContext } from 'react';
-import { ActivityIndicator, StyleSheet } from 'react-native';
+import React, { useCallback, useContext, useState } from 'react';
 import { Query } from 'react-apollo';
+import { ActivityIndicator, StyleSheet, TouchableOpacity, View } from 'react-native';
+import Carousel from 'react-native-snap-carousel';
 
-import { colors } from '../config';
+import { colors, Icon, normalize } from '../config';
 import { graphqlFetchPolicy, imageWidth, isActive, shareMessage } from '../helpers';
-import { getQuery } from '../queries';
-import { OrientationContext } from '../OrientationProvider';
-import { NetworkContext } from '../NetworkProvider';
 import { useRefreshTime } from '../hooks';
+import { NetworkContext } from '../NetworkProvider';
+import { OrientationContext } from '../OrientationProvider';
+import { getQuery } from '../queries';
+import { SettingsContext } from '../SettingsProvider';
 
 import { ImagesCarouselItem } from './ImagesCarouselItem';
 import { LoadingContainer } from './LoadingContainer';
 
 export const ImagesCarousel = ({ data, navigation, refreshTimeKey, aspectRatio }) => {
   const { dimensions } = useContext(OrientationContext);
   const { isConnected, isMainserverUp } = useContext(NetworkContext);
-
+  const { globalSettings } = useContext(SettingsContext);
+  const { settings = {} } = globalSettings;
+  const { sliderPauseButton } = settings;
+  const { show: showSliderPauseButton = false, size: sizeSliderPauseButton = 25 } =
```

Comment:
how come that the size configurable?

Reply:
When you change the size value in the sliderPauseButton object in the main server, the size of the button is updated according to the given value.

Reply:
yes that's the function. but i ask for why? why is that possible? who wants this?

Reply:
Unfortunately, I made this ticket without having a design, so it is a small precaution for any changes that may be required in the future.